config/config.php

```diff
@@ -210,66 +210,66 @@
 	'stable' => [
 		'13' => [
 			'100' => [
-				'latest' => '13.0.0',
-				'internalVersion' => '13.0.0.14',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-13.0.0.zip',
-				'web' => 'https://docs.nextcloud.com/server/12/admin_manual/maintenance/upgrade.html',
+				'latest' => '13.0.1',
+				'internalVersion' => '13.0.1.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-13.0.1.zip',
```

Comment:
@MorrisJobke why is 13.0.1 in prereleases and not releases?

Reply:
because it is not released yet. we are waiting for tomorrow for some more feedback on the RC.